server/src/main/java/org/schemarepo/server/RESTRepository.java

```diff
@@ -370,4 +351,15 @@ private SchemaEntry exists(SchemaEntry entry) {
     return entry;
   }
 
+  private Renderer getRenderer(String mediaType) {
+    // browsers usually send more than one type separated by comma
+    String primaryMediaType = mediaType == null ? null : mediaType.split(",")[0];
+    Renderer r = rendererByMediaType.get(primaryMediaType);
+    if (r == null) {
+      logger.info("No renderer configured for media type {}, defaulting to the renderer for {}", primaryMediaType, DEFAULT_MEDIA_TYPE);
```

Comment:
I think `415` would be too strict. Again, in tune of the original behavior, most of the time it's reasonable to assume that the client will be able to deal with the default-rendered (aka plain text) response. This is somewhat philosophical though.